tests/src/com/unciv/uniques/CountableTests.kt

```diff
@@ -510,9 +510,15 @@ class CountableTests {
     }
 
     @Test
-    @CoversCountable(Countables.RemainingCivs, Countables.FilteredUnits)
+    @CoversCountable(Countables.RemainingCivs, Countables.FilteredUnits, Countables.Carried)
     fun testFilteredUnitsCountable() {
-        setupModdedGame()
+        val ruleset = setupModdedGame()
+        val wetTile = game.tileMap[3,1]
+        wetTile.setBaseTerrain(ruleset.terrains[Constants.coast]!!)
+        val carrier = game.addUnit("Carrier", civ, game.tileMap[3,1])
+        val carried = game.addUnit("Fighter", civ, game.tileMap[3,1])
+        carried.isTransported = true
+        carrier.currentTile.airUnits.add(carried)
```

Comment:
Suspect. Why do those TestGame.addUnit functions require a Tile? The Fighter should already be in `airUnits` at this point. Heck, it was `isTransported` right after its `addUnit` ..if I am looking straight.